.cirrus.yml

```diff
@@ -46,7 +64,7 @@ low_scale_task:
 
   pack_caches_script:
     - rm -rf runtime-cache/*
-    - docker save -o runtime-cache/ovn-multi-node.tar ovn/ovn-multi-node:latest
+    - podman save -o runtime-cache/ovn-multi-node.tar ovn/ovn-multi-node:latest
     - tar -czf runtime-cache/runtime.tar.gz runtime
```

Comment:
Why we're saving the image again here?  It should be already in `runtime/ovn-fake-multinode/ovn-multi-node-image.tar`.

Reply:
You're right, it's there.  I had changed things so many times trying to figure out CI and in one of the versions I was (desperately) removing images from `runtime/..` to save disk space.  But that's not the case anymore.  I'll remove this and change the unpack_caches_script to load the image from the cached runtime.

Reply:
Done, seems to work:
https://cirrus-ci.com/task/5548665809403904?logs=unpack_caches#L2
https://cirrus-ci.com/task/6674565716246528?logs=unpack_caches#L2